src/backend/joanie/settings.py

```diff
@@ -237,7 +237,22 @@ class Base(Configuration):
             "COURSE_REGEX": values.Value(
                 r".*", environ_name="EDX_COURSE_REGEX", environ_prefix=None
             ),
-        }
+        },
+        {
+            "API_TOKEN": values.Value(
+                environ_name="MOODLE_API_TOKEN", environ_prefix=None
+            ),
+            "BACKEND": values.Value(environ_name="MOODLE_BACKEND", environ_prefix=None),
+            "BASE_URL": values.Value(
+                environ_name="MOODLE_BASE_URL", environ_prefix=None
+            ),
+            "SELECTOR_REGEX": values.Value(
+                r".*", environ_name="MOODLE_SELECTOR_REGEX", environ_prefix=None
+            ),
+            "COURSE_REGEX": values.Value(
+                r".*", environ_name="MOODLE_COURSE_REGEX", environ_prefix=None
+            ),
+        },
```

Comment:
In order to prevent conflicts, what do you think to change regexp that never match ?

Reply:
Please see the last commit.